config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     geoTiffResolution: 128,
     redirectLegacyUrls: false,
     itemsPerPage: 12,
+    maxDisplayPreviewsOnMapOnCatalogPage: 48,
```

Comment:
Maybe go with a slighly shorter name? Also, I think I'd reuse this option in the search component so removing the "OnCatalogPage" seems more future-proof.
Why exactly is it 48? That seems rather strange instead og e.g. 20, 50 or 100.
```suggestion
    maxPreviewsOnMap: 50,
```
This option also needs to be documented in the README.